examples/src/keyring/aws_kms/act_like_aws_kms_master_key_provider.py

```diff
@@ -0,0 +1,88 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+Before there were keyrings, there were master key providers.
+Master key providers were the original configuration structure
+that we provided for defining how you want to protect your data keys.
```

Comment:
This or line 4, but not both. They're redundant.
Also, avoid "we". Not prohibited, but it's ambiguous.
```suggestion
In earlier versions of the AWS Encryption SDK, you used master key providers to determine how your data keys are protected.
```